dist/index.js

```diff
@@ -8070,7 +8071,7 @@ async function generate(data) {
       );
       excludeLangs.push("python");
     } else if (isPullRequestEvent(githubEventName)) {
-      core.info("Detected a non-Forked PR, allowing all languages");
+      core.info("Detected no insecure configuration, allowing all languages");
```

Comment:
Normally, yes, but in this case I specifically mean we did not detect any
insecure configurations.

That doesn’t preclude any undetected insecure configurations because we
can’t really know that so rather than assert that it’s secure, I’m
asserting that we did not detect any known insecurities

Reply:
I get it!